ROADMAP.md

```diff
@@ -0,0 +1,123 @@
+xk6-browser roadmap
+===================
+
+xk6-browser is a young project that is undergoing substantial changes in order to make it more useful to all users—whether they're coming from API/performance testing with standard [k6](https://k6.io/), or are new to the k6 ecosystem and browser testing in general. In addition to being a useful tool, users would want to know what the team's, k6, and Grafana's goals are for the project, so that they can plan ahead and trust our commitment to its success.
```

Comment:
I think this intro is great. The only thing I would change is to remove the ~undergoing substantial changes~  because it may scare-off potential adopters. Most people read only ~25% of the document so we should start with the description of our ambitions rather than the current state which will change soon.

I would start with this: 
> xk6-browser is an extension that will become part of the k6 core once it reaches its stability goals. The project aims to add real-browser capabilities to k6 to make it more useful to all users—whether they're coming from (...)

Reply:
Makes sense. I rephrased your suggestion slightly and reworked the intro in 418abbd. Let me know.